README.md

```diff
@@ -108,25 +108,14 @@ Currently we don't have a playbook which installs KubeVirt on a Kubernetes clust
 
 ### OpenShift cluster
 
-When you have your cluster up and running you can install KubeVirt from a given manifest.
+Install KubeVirt on your OpenShift cluster
 
 ```bash
-$ # Get KubeVirt manifest
-$ wget https://github.com/kubevirt/kubevirt/releases/download/v0.2.0/kubevirt.yaml
-$ ansible-playbook -i localhost, --connection=local \
-        -e "openshift_ansible_dir=openshift-ansible/ \
-        kubeconfig=$HOME/.kube/config \
-        kubevirt_mf=kubevirt.yaml" \
-        playbooks/components/install-kubevirt-on-openshift.yml
+$ ansible-playbook -i localhost playbooks/kubevirt.yml -e @variables/all.yaml
 ```
-where
```

Comment:
Since you removed parameters from here, I would at least add link to KubeVirt's role doc where the parameters are described.

You can find read list of parameters at [KubeVirt role readme](./roles/kubevirt/README.md)

Or something like that.

Reply:
@rthallisey @lukas-bednar or smth like `Update default cluster parameters if needed.` And a link to `default` as roles/kubevirt/defaults/main.yml

And what do you think about adding the content of the current [KubeVirt role readme](https://github.com/kubevirt/kubevirt-ansible/pull/67/roles/kubevirt/README.md) directly into  roles/kubevirt/defaults/main.yml as comments? I think it will be handy to use it in that form.

Reply:
I added a link to the KubeVirt role params. The table contains the default values, which will be everything in ```roles/kubevirt/defaults/main.yml```